admin/controller/extension/payment/iyzico_checkout_form.php

```diff
@@ -0,0 +1,623 @@
+<?php
+
+error_reporting(E_ALL);

```

Comment:
bu error reporting'in kapalı olması gerekiyor. server ile ilgili hassas bilgileri ekrana basma riski var.